### ISSUE TITLE
Moving a token that is hidden makes it visible

### ISSUE THREAD

**Issue by feltz916** [gludington/conditional-visibility#24]
Pretty sure this started after 0.7.5 update.

Steps to repro:
- Roll stealth on a npc token
- See it disappear on player screen
- move the npc token
- see it appear on player screen

**Comment by gludington** [comment]
It appears on token move, the rendering goes through a different code path altogether until the next time the token doing the seeing is touched, which means this module does not touch it.  Will need to figure out where that separate drawing pipeline is, and see if it can be inserted at all.

**Comment by gludington** [comment]
0.75 introduced a different code path for moving tokens, such that monkeypatching it for a module could be dangerously brittle.  Filed an issue for discussion at https://gitlab.com/foundrynet/foundryvtt/-/issues/3974

**Comment by feltz916** [comment]
@gludington it looks like some support for this was put into 0.7.6 core, will that allow progress to be made?

**Comment by gludington** [comment]
Yes, it does, and I have been looking at it as time permits, though I may not be able to fully commit until the weekend.  If you are interested/able to install from and test a pre-release manifest URL, I may be able to place a work in progress tomorrow after work.

**Comment by Avitale34** [comment]
Still having this issue :( any chance of fix? I love using this module!

**Comment by OmnesPotens** [comment]
@gludington This seems to be happening with the "In Darkness" condition as well.

Foundry version: 0.7.9
5e Version: 1.2.0

**Comment by feltz916** [comment]
any updates to this issue?

**Comment by ArdonDaros** [comment]
This problem is also manifest with PF1 system, running the latest stable versions of both Foundry (0.7.9) and PF1 system: The token is correctly hidden, but as soon as the affected token is moved, players are able to see it normally.
